recipes-ni/ni-systemformat/ni-systemformat.bb

```diff
@@ -25,21 +34,18 @@ do_install () {
 	install -d ${D}${sysconfdir}/init.d
 	install -m 0755 ${S}/nitargetinfo ${D}${sysconfdir}/init.d/
 
-    # legacy symlink location
-    install -d ${D}/usr/local/natinst/bin
-    ln -sf ${bindir}/nisystemformat ${D}/usr/local/natinst/bin/nisystemformat
+	# legacy symlink location
+	install -d ${D}/usr/local/natinst/bin
+	ln -sf ${bindir}/nisystemformat ${D}/usr/local/natinst/bin/nisystemformat
 }
 
 
-FILES_${PN} += "\
+FILES:${PN} += "\
 	${bindir}/nisystemformat \
 	${sysconfdir}/init.d/nitargetinfo \
 	/usr/local/natinst/bin/nisystemformat \
 "
-
-inherit update-rc.d
-
-INITSCRIPT_NAME = "nitargetinfo"
-INITSCRIPT_PARAMS = "start 20 S ."
-
-RDEPENDS_${PN} += "bash"
+RDEPENDS:${PN} += " \
+	bash \
+	niacctbase \
```

Comment:
Why have you moved this from the DEPENDS list? Your commit description does not say why.